### ISSUE TITLE
Add unit tests for CI

### ISSUE THREAD

**Issue by lqy0057** [decisionforce/pgdrive#173]
To prevent new PR from introducing new bug, we need some unit tests to do CI

**Comment by pengzhenghao** [comment]
The minimal unit test cases I think that are necessary to be introduced are: **A set of blackbox tests that are the grid search of all possible combination of config that can be used to built a no-rendering PGDrive env.**

The term `blackbox test` here means a basic testing:

(pseudo code)
```
env = PGDrive(config)
obs = env.reset()
assert obs in env.obs_space
o, r, d, i = env.step(env.action_space.sample())
assert o in env.obs_space
assert r in reward range
assert i have keys: xxxx
env.reset()
env.close()
```

So that we can at least make sure the environment can be successfully instantiated, step, reset and close.

**Comment by lqy0057** [comment]
Solved in #198
